x-pack/plugins/cases/common/api/cases/case.ts

```diff
@@ -244,6 +244,16 @@ export const CaseResponseRt = rt.intersection([
   }),
 ]);
 
+export const CaseResolveResponseRt = rt.intersection([
+  rt.type({
+    saved_object: CaseAttributesRt,
```

Comment:
Thoughts on changing `saved_object` to `case`? Saved object might be a little misleading because the contents might not be a fully valid SavedObject type.